docs/eventing/broker/kafka-configmap.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Kafka Channel ConfigMap"
+weight: 30
+type: "docs"
+---
+
+**NOTE:** This guide assumes Knative Eventing is installed in the `knative-eventing`
```

Comment:
I suggest adding this note to the config-br-defaults.md file, since it also mentions the `knative-eventing` namespace.